step-definitions/Hotels/HotelsSD.js

```diff
@@ -0,0 +1,213 @@
+const { Given, When, Then } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const Feedbackpage = require("../../Pages/Hotels/Feedbackpage");
+const Homepage = require("../../Pages/Hotels/Homepage");
+const fpage = new Feedbackpage();
+const hpage = new Homepage();
+
+Given(/^I am on hotels landing page$/, async function(){
+    await browser.url('/');
+    await browser.pause(2000);
+});
+When(/^I click on Travelers$/, async function(){
+    await hpage.clickOnTravelers();
+    await browser.pause(2000);
+});
+Then(/^I select adults as "6"$/, async function(){
```

Comment:
All Action step => When
All Verification step => Then
Pls. correct for all SD.